Makefile

```diff
@@ -259,6 +259,7 @@ ifeq ($(INSTALL_COQDEV),true)
           install -d $(COQDEVDIR)/$$d && \
           install -m 0644 $$d/*.vo $(COQDEVDIR)/$$d/; \
 	done
+	cp ./VERSION $(COQDEVDIR)
 	@(echo "To use, pass the following to coq_makefile or add the following to _CoqProject:"; echo "-R $(COQDEVDIR) compcert") > $(COQDEVDIR)/README
```

Comment:
Please use "install -m 0644" instead of "cp".  This is what every other installation action does in this Makefile, and it's important to get permissions right.